lin_actuator.h

```diff
@@ -0,0 +1,36 @@
+/* 
+ * File:   LINAC.h
+ * Author: zx100
+ *
+ * Created on February 10, 2019, 12:25 AM
+ */
+
+#ifndef LINAC_H
+#define	LINAC_H
+
+typedef enum
```

Comment:
note: enums in c increment automatically, you don't need to assign a value to each one.